cmd/flowrunner/main.go

```diff
@@ -232,7 +231,7 @@ func main() {
 		scanner.Scan()
 
 		// create our event to resume with
-		msg := inputs.NewMsgInput(flows.InputUUID(utils.NewUUID()), nil, time.Now(), contact.URNs()[0].URN, scanner.Text(), []flows.Attachment{})
+		msg := flows.NewMsgIn(flows.MsgUUID(utils.NewUUID()), contact.URNs()[0].URN, nil, scanner.Text(), []flows.Attachment{})
```

Comment:
Seems nicer to just add a `NewMsgUUID` function here..

Reply:
Agree and I also think it would be nice to move those UUID subtype definitions closer to the structs that use them rather than having a list in interfaces.go But we do this everywhere in goflow now so maybe can change in different PR.